caffe2/CMakeLists.txt

```diff
@@ -476,6 +476,22 @@ if (NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
     ${TORCH_SRC_DIR}/csrc/jit/tensorexpr/tensor.cpp
     )
 
+  if (USE_LLVM)
```

Comment:
@ZolotukhinM I pulled the llvm cmake changes from your cmake integration branch but I left almost everything else out to keep this diff small.  Just a heads up when you're rebasing.